File: models/budget.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace budgetmanagementAngular.models
{
    public class budget
    {
        public int budgetID { get; set; }
        public int month { get; set; }
        public int year { get; set; }
        public decimal totalIncome { get; set; }
        public ICollection<budgetCategory> categories { get; set; }

    }
}

File: ClientApp/src/app/budget/classes/budget.ts
export class Budget {
  budgetID: number = -1;
  month: number = new Date().getMonth() + 1;
  year: number = new Date().getFullYear();
  totalIncome: number=0;
  totalSpent: number = 0;
  totalLeftToBudget: number = 0;
}
